src/main/java/hudson/plugins/clover/CloverBuildAction.java

```diff
@@ -149,10 +156,20 @@ static CloverBuildAction getPreviousResult(Run<?, ?> start) {
         if (c != null) {
             c.setOwner(build);
         }
+        if( this.projectActions == null )
```

Comment:
why these null checks if the only place where field is assigned is the CloverBuildAction constructor and I don't see any place where this field is set to null? it it because of object serialisation for remote agents?